packages/vue-app/template/App.js

```diff
@@ -67,10 +83,12 @@ export default {
     // add to window so we can listen when ready
     if (process.client) {
       window.<%= globals.nuxt %> = <%= (globals.nuxt !== '$nuxt' ? 'window.$nuxt = ' : '') %>this
+      <% if (features.client.online) { %>
```

Comment:
Funny fact: As a nuxt developer, wasn't aware we have it 😂😂😂

Reply:
It came together with smart prefetching :relaxed: